.github/workflows/homebrew.yml

```diff
@@ -0,0 +1,19 @@
+name: update-tap
+
+on:
```

Comment:
This action will run on every push. Is that what you intended?

Shouldn't this be

```
on:
  release:
    types: [created]
```